src/libs/ajax.js

```diff
@@ -495,8 +495,8 @@ const Buckets = signal => ({
 
       getObject,
 
-      rename: async newName => {
-        await copy(newName)
+      rename: async (newName) => {
```

Comment:
Tiny thing, remove parens for single argument, thanks!

Reply:
Good catch, thanks.